src/Clearinghouse.sol

```diff
@@ -68,10 +69,11 @@ contract Clearinghouse is Policy, RolesConsumer, CoolerCallback {
     /// @notice timestamp at which the next rebalance can occur.
     uint256 public fundTime;
 
-    /// @notice outstanding loan receivables.
+    /// @notice Outstanding receivables.
     /// Incremented when a loan is taken or rolled.
     /// Decremented when a loan is repaid or collateral is burned.
-    uint256 public receivables;
+    uint256 public interestReceivables;
+    uint256 public principleReceivables;
```

Comment:
"principal", not "principle." This misspelling occurs throughout.

Reply:
we fixed it in the follow-up PRs